mailpoet/assets/js/src/automation/editor/components/automation/types.ts

```diff
@@ -7,6 +7,7 @@ export type NextStep = {
 export type Step = {
   id: string;
   type: 'root' | 'trigger' | 'action';
+  subject_keys: string[];
```

Comment:
Why add it here? It seems to me that this data doesn't belong in the step instance, only to the [registry as you did below](https://github.com/mailpoet/mailpoet/pull/4737/commits/7fb717f89495b166a88ba0d3e534ae4ad222ad01#diff-cb1312bbc63f2b97cea4601c15449850fb79af2ea6b93a30137a6734cdd7a439R15). Could we use it only from there?

Reply:
Thank you for pointing that out! 1f3e94b3d